test/Microsoft.Azure.Devices.Edge.Test/Device.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common;
+    using Microsoft.Azure.Devices.Edge.Test.Common.Config;
+    using Microsoft.Azure.Devices.Edge.Test.Helpers;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using NUnit.Framework;
+
+    class Device : DeviceBase
+    {
+        [Test]
```

Comment:
Currently all tests are in EndToEnd.cs; and Device class is created which contains tests.  What do you expect for new tests? what file structure should be looked like?

Reply:
`EndToEnd.cs` was removed in this PR. Tests related to modules (currently: temp sensor, module-to-module direct methods) are in `Module.cs`. Tests related to devices (currently: transparent gateway) are in `Device.cs`.